### PR TITLE
Use access_token grant instead of refresh_token

### PR DESCRIPTION
**Pull request by bwateratmsft**
This substantially mitigates #1959 or possibly eliminates the issue entirely. Changing the ADAL token <=> ACR token exchange's `grant_type` to `access_token`, and not passing in the ADAL `refresh_token`, dramatically reduces the size of the obtained ACR `refresh_token`. My token's size was pretty consistently about 950 bytes, compared to > 2600 before.

It may be functionally impossible to get a token using this different mechanism that is too long for Docker CLI. We won't update people's existing Docker configs (at least not now...), but we hopefully will never need to suggest disabling Wincred again.

The disadvantage to this grant type is that if the ADAL `access_token` is near expiry, they (ACR) will not be able to use the ADAL `refresh_token` to get a new one--however, that shouldn't be an issue for us since we get a fresh ADAL token from the Azure Account extension every time.

### CODE REVIEW

File: src/commands/context/aci/createAciContext.ts [hunk 1]
```diff
@@ -4,7 +4,7 @@
  *--------------------------------------------------------------------------------------------*/
 
 import { Progress } from 'vscode';
-import { AzureWizard, AzureWizardExecuteStep, AzureWizardPromptStep, IActionContext, IResourceGroupWizardContext, LocationListStep, parseError, ResourceGroupListStep } from 'vscode-azureextensionui';
```
Comment (karolz-ms) [inline src/commands/context/aci/createAciContext.ts:7]: This file has unrelated changes--please exclude

Comment (bwateratmsft) [reply src/commands/context/aci/createAciContext.ts:7]: I was gonna put them together (me lazy) but I can separate.
